unstract/connectors/src/unstract/connectors/databases/mysql/mysql.py

```diff
@@ -78,3 +79,13 @@ def sql_to_db_mapping(value: str) -> str:
             datetime.datetime: "TIMESTAMP",
         }
         return mapping.get(python_type, "LONGTEXT")
+
+    def execute_query(
```

Comment:
Why it not returning anything?

Reply:
@muhammad-ali-e this will handle run the insert query. No need to return